src/Homework_02/03_Max_Dlina.py

```diff
@@ -0,0 +1,28 @@
+"""
+3. Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.
+"""
+
+my_string = 'aaa bbbb! aa, bbb! c bbbaa bb b. b? aaa'
+print(my_string, '\n')
+
+# не самый красивый вариант
+new_string = my_string.replace(
```

Comment:
what about ?(){} etc? 
you can use string with punctuation characters and using for loop replace all of them

Reply:
Fixed